rdr_client/hpo_assigner.py

```diff
@@ -98,4 +99,5 @@ def _get_old_hpo(participant):
   arg_parser = get_parser()
   arg_parser.add_argument('file', help='file containing the list of HPOs and participant IDs')
   arg_parser.add_argument('--dry_run', action='store_true')
+  arg_parser.add_argument('--pairing', help='set level of pairing at site')
```

Comment:
It looks like this is required above. If so, you may want to add required=True here. Alternatively, make it default to awardee.

Reply:
done